SmartOrder.csproj

```diff
@@ -74,13 +81,24 @@
       WorkingDirectory="$(MSBuildProjectDirectory)" 
     />
   </Target>
-
+  
+  <!-- Normal build process -->
   <Target
-    Name="Build TailwindCSS" 
-    AfterTargets="Build"
+    Name="Build TailwindCss"
+    BeforeTargets="PreBuildEvent"
+    DependsOnTargets="FindTailwindCss"
+    Condition="'$(DotNetWatchBuild)' != 'true'"
   >
-    <Exec 
-      Command="$(ProjectDir)build-tailwind.sh" 
-    />
+    <Exec Command="$(TailwindCssPath) build -i ./Styles/site.css -o ./wwwroot/css/site.css" />
+  </Target>
+  
+  <!-- run tailwindcss watch process by using `dotnet watch run` (or just `dotnet watch`)-->
+  <Target
+    Name="Watch TailwindCss"
+    BeforeTargets="PreBuildEvent"
+    DependsOnTargets="FindTailwindCss"
+    Condition="'$(DotNetWatchBuild)' == 'true'"
+  >
+    <Exec Command="$(TailwindCssPath) build -i ./Styles/site.css -o ./wwwroot/css/site.css --watch" />
   </Target>

```

Comment:
That's true, we can definitely do that.  Especially if the tailwind startup is really fast.  If there's a lot of hang time for a large set of styles, then it might be better to keep it as a long running process, sort of like they do for Typescript

Reply:
We can even limit it to only run when the styles themselves change, by adding the `Inputs` parameter to the `Target`.  That way, it doesn't always have to run tailwind, any time any of the source files changes.

Reply:
I suppose this will also kill the tailwind watch automatically when the parent watch is killed. Lol and I was trying to do that manually...

Considering that tailwind watch will be better tuned for only rebuilding on style changes, let's do it your way.